csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/InferenceTest.netcore.cs

```diff
@@ -1,12 +1,19 @@
 ﻿using Microsoft.ML.OnnxRuntime.Tensors;
+using Microsoft.VisualStudio.TestPlatform.Utilities;
```

Comment:
>using Microsoft.VisualStud [](http://example.com/codeflow?start=0&length=26)

Can we ensure that usings are sorted? VS has a feature to do it.

Reply:
Fixed.